lms/envs/common.py

```diff
@@ -1257,7 +1257,7 @@
     'js/vendor/url.min.js',
     'common/js/vendor/underscore.js',
     'common/js/vendor/underscore.string.js',
-    'common/js/vendor/picturefill.min.js',
```

Comment:
You want to make this change in dev environment only, right?
I believe the common.py is being used in production environment and there are no overrides for other environment files. This means with your change here, the picturefill library with no minification will be used in prod also. Is this what we want?